src/server/components/dashboard/listsItems/getViewModel.ts

```diff
@@ -170,25 +192,46 @@ function getOrganisationRows(listItem: ListItemGetObject): Types.govukRow[] {
       "repatriationServicesProvided",
       "religiousCulturalServicesProvided",
     ],
+    [ServiceType.translatorsInterpreters]: [
+      ...baseFields,
+      "servicesProvided",
+      "languagesProvided",
+      "languagesSummary",
+      "translationSpecialties",
+      "interpreterServices",
+      "deliveryOfServices",
+      "representedBritishNationals",
+    ],
   };
 
   const fieldsForType = fields[type] ?? baseFields;
 
+  if (type === ServiceType.translatorsInterpreters && listItem.jsonData.deliveryOfServices) {
+    listItem.jsonData.deliveryOfServices = DeliveryOfServices[listItem.jsonData.deliveryOfServices];
+  }
+  if (type === ServiceType.translatorsInterpreters && listItem.jsonData.languagesProvided) {
+    const languagesArray = listItem.jsonData.languagesProvided.map((item: string) => {
+      return languages[item] || item;
+    });
```

Comment:
Really small optional change but if you're just returning one line in an arrow function you can get rid of the braces and the `return` keyword.

```suggestion
    const languagesArray = listItem.jsonData.languagesProvided.map((item: string) => languages[item] || item);
```

Reply:
yep, makes sense.  Change made.